R/spaces_inside_linter.R

```diff
@@ -1,7 +1,8 @@
 #' Spaces inside linter
 #'
-#' Check that parentheses and square brackets do not have spaces directly inside them, i.e., directly following an
-#' opening delimiter or directly preceding a closing delimiter.
+#' Check that parentheses and square brackets do not have spaces directly
+#'   inside them, i.e., directly following an opening delimiter or directly
+#'   preceding a closing delimiter.
```

Comment:
Is there a reason for adhering to line length < 120?

Reply:
my emacs not wide enough and I'm too noob to figure out how to adjust 😢

but more generally I think of our 120 as an "if needed for readability feel free to spill over 80" guideline

Reply:
Ah I always use the full 120 columns because my editor windows are pretty wide 😄  [SO Seems to have something for you](https://stackoverflow.com/questions/92971/how-do-i-set-the-size-of-emacs-window)

Reply:
thanks for googling for me :laughing: might as well be Lao to me though